controllers/topics/checkName.js

```diff
@@ -0,0 +1,37 @@
+const {Topics} = require('../../databases/models');
+
+module.exports = async (req, res) => {
+  try {
+    let {name} = req.body;
```

Comment:
@brm-stnd @ibramastana 
related to my comment below, please use `req.query` instead of body since `router.get` can't use `req.body`

Reply:
fixed